k9mail/src/main/java/security/planck/passphrase/extensions/String.kt

```diff
@@ -0,0 +1,9 @@
+package security.planck.passphrase.extensions
+
+private const val ACCEPTED_SYMBOLS = """@\$!%*+\-_#?&\[\]\{\}\(\)\.:;,<>~"'\\/"""
+private const val PASSPHRASE_REGEX =
```

Comment:
do we have a BSI rule for this? 12 min?

Reply:
Yes, this was defined in EFA-560 ticket, just moved here since we don't use old Presenter anymore.

Reply:
perfect